src/google-maps/google-map/google-map.ts

```diff
@@ -35,7 +35,8 @@ interface GoogleMapsWindow extends Window {
 /** default options set to the Googleplex */
 export const DEFAULT_OPTIONS: google.maps.MapOptions = {
   center: {lat: 37.421995, lng: -122.084092},
-  zoom: 17
+  zoom: 17,
+  mapTypeId: 'roadmap'
```

Comment:
Getting this error internally:
```
TS2322: Type 'string' is not assignable to type 'MapTypeId | undefined'.
```

Could we instead do `'roadmap' as MapTypeId` to resolve that?

Reply:
Done, although it's weird that our CI didn't catch it.

Reply:
Yeesh now I'm getting this

```
TS2352: Conversion of type 'string' to type 'MapTypeId' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```

Reply:
Done.